addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/handler/WebserviceHandler.java

```diff
@@ -139,6 +139,13 @@ public void updateStationData() {
                 // if the result is not OK, no updates are done and the status of the Bridge goes to OFFLINE!
                 updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
                         "Empty return or no internet connection");
+                // if the Bridge goes OFFLINE, all Stations will go OFFLINE as well
+                for (Thing thing : getThing().getThings()) {
+                    StationHandler tkh = (StationHandler) thing.getHandler();
+                    tkh.updateStatus(ThingStatus.OFFLINE);
```

Comment:
I might be wrong, but for me looked as if only the bridge goes OFF LINE with the current code.
I do not know of any other "smart"way to do this, but I'm not that much about ESH.